docs-source/content/userguide/managing-fmw-domains/soa-suite/_index.md

```diff
@@ -0,0 +1,333 @@
+
+```
+$ docker tag oracle/fmw-infrastructure:12213-update-k8s container-registry.oracle.com/middleware/fmw_infrastructure:12.2.1.3
+```
+
+After cloning the repository and downloading the installer from Oracle Technology Network
+or e-delivery, you create your image by running the provided script:
+
+```bash
+cd docker-images/OracleFMWInfrastructure/dockerfiles
+./buildDockerImage.sh -v 12.2.1.3 -s
+```
+
+The image produced will be named `middleware/soasuite/oracle/soasuite:12.2.1.3`.
+
+The Oracle SOA Suite image created through the above step needs to be retagged
+from `middleware/soasuite/oracle/soasuite:12.2.1.3` to `oracle/soa:12.2.1.3` before continuing with the next steps.
+
+```bash
+$ docker tag middleware/soasuite/oracle/soasuite:12.2.1.3 oracle/soa:12.2.1.3
+```
+
+You can use this image to run the Repository Creation Utility and to run your domain using the “domain on a persistent volume” model.
+
+Before creating a domain, you will need to set up the necessary schemas in your database.
+
+#### Configuring access to your database
+
+SOA Suite domains require a database with the necessary schemas installed in them.
+A utility called Repository Creation Utility (RCU) is provided which allows you to create
+those schemas.  You must set up the database before you create your domain.
+There are no additional requirements added by running SOA in Kubernetes; the
+same existing requirements apply.
+
+For development or test environments, you may use the container-based database,
+whereas for production environments, it is recommended that you use the standalone database.
+
+The following documentation and samples are for creating a container-based database on a persistent volume.
+
+##### Running the database inside Kubernetes
+
+Follow these instructions for a basic database setup inside Kubernetes that uses PV (persistent volume) and PVC (persistent volume claim) to persist the data. For more details about database setup and configuration, refer this [page](https://oracle.github.io/weblogic-kubernetes-operator/userguide/managing-domains/fmw-infra/#running-the-database-inside-kubernetes).
+
+Pull the database image:
+
+```bash
+$ docker pull container-registry.oracle.com/database/enterprise:12.2.0.1
+$ docker tag  container-registry.oracle.com/database/enterprise:12.2.0.1  oracle/database:12.2.0.1
+```
+Create the PV and PVC for the database:
+
+```bash
+$ mkdir /scratch/DockerVolume/DB
+$ chmod 777 /scratch/DockerVolume/DB
+```
+Update the appropriate host/storage path in `db-pv.yaml`.
+
+Create the PV/PVC using these commands:
+
+```bash
+$ cd  weblogic-kubernetes-operator/kubernetes/samples/scripts/create-database-pv-pvc
+$ kubectl create -f db-pv.yaml
+$ kubectl create -f db-pvc.yaml
+```
+Bring up the database and database service using the following commands:
+
+```bash
+$ cd soa-kubernetes-operator/kubernetes/samples/scripts/create-soa-domain
```

Comment:
Please change this line
cd soa-kubernetes-operator/kubernetes/samples/scripts/create-soa-domain
to 
cd weblogic-kubernetes-operator/kubernetes/samples/scripts/create-soa-domain/create-database